MediaBrowser.Providers/MediaInfo/VideoImageProvider.cs

```diff
@@ -88,30 +88,14 @@ public async Task<DynamicImageResponse> GetVideoImage(Video item, CancellationTo
 
             if (imageStream != null)
             {
-                // Instead of using the raw stream index, we need to use nth video/embedded image stream
```

Comment:
This is the logic that would cause us to have the correct behavior for video files. The video index is computed as the "Nth video stream".

By using the absolute index, we no longer need to perform this calculation.